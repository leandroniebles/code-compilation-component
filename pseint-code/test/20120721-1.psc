Proceso sin_titulo
	a<-2; b<-3; c<-4;
	si a Entonces
		Escribir "lalA";
	FinSi
	mientras b hacer
	  escribir "lala";
	finmientras
	repetir
	  escribir "lala";
	hasta que c
FinProceso
