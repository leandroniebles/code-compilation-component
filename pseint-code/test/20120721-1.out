=== Line 3: SynError 92
=== Line 6: SynError 104
=== Line 11: SynError 95
